src/sentry/api/endpoints/organization_group_index_stats.py

```diff
@@ -16,7 +17,7 @@
 
 
 @region_silo_endpoint
-class OrganizationGroupIndexStatsEndpoint(OrganizationEventsEndpointBase):
+class OrganizationGroupIndexStatsEndpoint(OrganizationEndpoint):
```

Comment:
Does not look like this class needs to extend `OrganizationEventsEndpointBase`
extending `OrganizationEndpoint` instead